src/biome/text/pipeline.py

```diff
@@ -418,20 +426,19 @@ def type_name(self) -> str:
     @property
     def trainable_parameters(self) -> int:
         """
-        Return the number of trainable parameters.
-
-        This number could be change before an after a training process, since trainer could fix some of them.
+        Returns the number of trainable parameters.
 
+        Keep in mind that this number also depends on the configuration of your trainer.
```

Comment:
At training time, this number can change when freezing/unfreezing certain parameter groups.

(internal note: although I think this is not possible at the moment)

Reply:
Done